### ISSUE TITLE
Enhancement: Admin setting for castle heart

### ISSUE THREAD

**Issue by Asuka-Tech** [Odjit/KindredLogistics#12]
i love this plugin, it would be a nice idea to have a admin settings, default off, that alows the conveyor system to store blood essence in the castle heart, castle heart cant be renamed so on default the conveyor system dosnt apply, it be great to have the addition to enable a option that allows it to interact/store with the castle heart.

**Comment by mfoltz** [comment]
Check out the '.freezeheart' command from KindredCommands (https://thunderstore.io/c/v-rising/p/odjit/KindredCommands/) if you haven't seen that one yet, might be what you're looking for here.

**Comment by Asuka-Tech** [comment]
yeah i know this one, but i like the idea of my castle heart requiring blood to keep my castle up.
even if it would be automated and it eats from my storage.

**Comment by Odjit** [comment]
This is something I've considered before. 

So far I haven't done it because it is a small use case. 

Naturally the game allows you to get I think like 60 days? Off of what you can load on to the heart itself. Furthermore having it feed into the castles would turn a lot of features on tracking time left on castles moot. Granted this would all be self-inflicted problems by any admin who would turn it on but again it reduces the use case.

I can see a slim use where you want the fuel consumption to be high and stacks low but yet want people to be able to keep their castles a long time. It's just a weird thing where it's like you for some reason don't want to give them the high stack and low fuel consumption but still want to give them time. Which is odd because that's what those two things do for you. 

Basically this would all be adding more things for the server to have to move around for no particularly good reason when there are other options that will achieve essentially the same desire except not use performance to do so. 

All that said, if there is something I haven't considered about it, let me know!

**Comment by Asuka-Tech** [comment]
yeah i get the points. i just like the idea of automation, not having to check the heart as long as i play every now and then. simply using the stash feature to insert it into the system.
its really just simplicity at least for me, while still having decay.
as said it could be a setting for admins to allow but is default off.

**Comment by mfoltz** [comment]
closing for now to keep issues tidy.
